blueflood-core/src/main/java/com/rackspacecloud/blueflood/service/Configuration.java

```diff
@@ -81,36 +95,50 @@ public int getIntegerProperty(Enum<? extends ConfigDefaults> name) {
         return getIntegerProperty(name.toString());
     }
```

Comment:
Well, all those three methods only have "core java" code in it. If we are testing those methods, we are only testing if java does what it is supposed to do. But if you have plans to add some application logic later on, thats fine. Also they dont have to be static.